src/createServer.js

```diff
@@ -1,3 +1,38 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { getErrors } = require('./getErrors');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const currentURL = new URL('htto://' + req.url);
+    const reqText = currentURL.pathname.slice(1);
+    const reqCase = currentURL.searchParams.get('toCase');
```

Comment:
You can name these variables as 'originalText', 'targetCase' and avoid unnecessary code in lines 22-23.